src/dimred/lsi/config.vsh.yaml

```diff
@@ -0,0 +1,117 @@
+functionality:
+  name: lsi
+  namespace: "dimred"
+  description: |
```

Comment:
@SarahOuologuem , do you also want to put yourself in authors? :) You can find an example here: https://github.com/openpipelines-bio/openpipeline/blob/b41a65886f61959c7f83f84a0569c3da1d7a2856/src/neighbors/find_neighbors/config.vsh.yaml#L9